readme.md

```diff
@@ -13,3 +13,5 @@ login page design updates
 fix auth redirect error
 
 new TOS page
+
+new mention handler tests
```

Comment:
this is cool @botany-demo

Reply:
check this out @botany-dummy-org-1/team_1

Reply:
seriously great @botany-demo, please look @botany-dummy-org-1/team_1

Reply:
thanks so much @botany-lead!

Reply:
thanks so much @botany-lead!

Reply:
and thank you @botany-dummy-org-1/team_1

Reply:
hey there @botany-demo

Reply:
hey there @botany-dummy-org-1/team_1